### ISSUE TITLE
replication_slots is broken when there is no logical replication slot

### ISSUE THREAD

**Issue by anayrat** [OPMDG/check_pgactivity#202]
Hello,

It seems replication_slots  check is broken here : https://github.com/OPMDG/check_pgactivity/blob/602ae71f4caa0a169ea9ba2a8ba63ea6230fb4b7/check_pgactivity#L5811

If there is no file in pg_replslot `$row->[3]` is empty and you go this error : 
```
Argument "" isn't numeric in numeric gt (>) at /opt/sensu-plugins/check_pgactivity line 5811.
```
I will try to address this issue soonly.

Regards,

**Comment by anayrat** [comment]
I investigate and this occurs before 10 due to `null as replslot_files`:
https://github.com/OPMDG/check_pgactivity/blob/90796a14752c9e830241a09766d41f6ed3df9560/check_pgactivity#L5703

We could change to 0 but I don't remember why we do not count replslot_files before 10? We could have spill files since 9.4 if we use logical replication slot (even without native logical replication) ?

So I see to solutions:
  * Replace null by 0 in 9.4's query
  * Count replslot_files in 9.4

First fix is quick, second not :)

**Comment by rjuju** [comment]
@anayrat isn't that a change you introduced in https://github.com/OPMDG/check_pgactivity/pull/176?  I didn't follow that PR so I have no idea why the slot files aren't checked before 10.  Can you check if you can get this metric for pg9.4+ too?

**Comment by anayrat** [comment]
You're right, it seems I added logical replication slot support since 10 due to native logical replication appeared in 10. But, as I mentioned in this PR, spill files can appear since 9.4. I'll work on this.

**Comment by rjuju** [comment]
Thanks!

**Comment by ioguix** [comment]
What up here? @anayrat, will fix your stuff? :)

**Comment by anayrat** [comment]
Sorry, I forgot this issue :(

I do not have time to spend on this. I think, the best that could be done is to count replslot_files in 9.4.

**Comment by ioguix** [comment]
So maybe this check should be set as compatible with 10+ and mark this issue as a futur enhancement? If it makes sense and we deal with it before 9.6 is deprecated :D

I'm not sure when someone will have time to spend on this...

**Comment by ioguix** [comment]
Fixed in PR #273
